components/event-bus/internal/knative/subscription/controller/knativesubscription/reconcile_test.go

```diff
@@ -1,30 +1,29 @@
 package knativesubscription
 
 import (
+	"context"
 	"fmt"
+	"testing"
 	"time"
 
-	"k8s.io/apimachinery/pkg/runtime"
-	"sigs.k8s.io/controller-runtime/pkg/reconcile"
-
-	"knative.dev/pkg/apis"
-
-	controllertesting "github.com/knative/eventing/pkg/reconciler/testing"
-	eventingv1alpha1 "github.com/kyma-project/kyma/components/event-bus/api/push/eventing.kyma-project.io/v1alpha1"
-	subApis "github.com/kyma-project/kyma/components/event-bus/api/push/eventing.kyma-project.io/v1alpha1"
+	kymaeventingv1alpha1 "github.com/kyma-project/kyma/components/event-bus/apis/eventing/v1alpha1"
+	fakeeventbusclient "github.com/kyma-project/kyma/components/event-bus/client/generated/injection/client/fake"
+	. "github.com/kyma-project/kyma/components/event-bus/internal/knative/subscription/controller/testing"
 	"github.com/kyma-project/kyma/components/event-bus/internal/knative/util"
-	duckv1beta1 "knative.dev/pkg/apis/duck/v1beta1"
-	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
-	//	"github.com/kyma-project/kyma/components/event-bus/internal/knative/subscription/controller/subscription"
-	"testing"
-
-	evapisv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	evapisv1alpha1 "knative.dev/eventing/pkg/apis/eventing/v1alpha1"
```

Comment:
(nit) `eventingv1alpha1` would be better aligned with the other packages aliases. (or `kneventingv1alpha1` to avoid mixing with Kyma's eventing group)